Commands/GenerateVisits.php

```diff
@@ -35,6 +36,14 @@ protected function configure()
         $this->addOption('no-logs', null, InputOption::VALUE_NONE, 'If set, no visits from logs will be generated', null);
         $this->addOption('limit-fake-visits', null, InputOption::VALUE_REQUIRED, 'Limits the number of fake visits', null);
         $this->addOption('custom-piwik-url', null, InputOption::VALUE_REQUIRED, "Defines an alternate Piwik URL, e.g. if Piwik is installed behind a Load-Balancer.");
+        $this->addOption('from-now', null, InputOption::VALUE_REQUIRED,
```

Comment:
@diosmosis from what I see, when entering eg 300 seconds, we replay the requests for the next 5 minutes at once correct? Would it be possible without too much work, that when we replay "live", to fetch the log lines for the next 300 seconds, log the first request, then calculate the time difference between this and the next log line. For example, if in the log line the next request was made 10 seconds later, could we sleep for 10 seconds before replaying the next line? This would make it even more "live" if you know what I mean? Hope I described it well. It is not a must, would just make it even more realistic when looking at the real time widgets.

Reply:
I see what you mean, and I think it would be kind of hard w/ PHP to really schedule requests to go out at specific times (at least I'm not aware of an easy way to do it).

It would be super simple w/ node, though, but I think that's not an option?

Reply:
Alternatively, we could run a loop in smaller intervals, eg:

```
$interval = 30;
$totalFromNow = 300;
for ($i = 0; $i < $totalFromNow; $i += $interval) {
    trackVisitsWithFromNow($interval);
    sleep($interval);
}
```

so every 30s, we track visits in the last 30s or something.

Reply:
Not sure if that last idea will be an issue given how large the files could be (collectively).

Reply:
If we require there be a single log file w/ dates in order, then the interval idea would work, I think.

Reply:
I'll just make the required changes.

Reply:
We can definitely create one log file from all the log files. And just running in a loop in smaller intervals would work as well 👍